source/hosting/migrate-netlify.txt

```diff
@@ -0,0 +1,196 @@
+.. _migrate-netlify:
+
+Migrate Static Hosting to Netlify
+=================================
+
+.. meta::
+  :description: Learn how to migrate your static assets from Atlas App Services to Netlify.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and `Netlify 
+   <https://docs.netlify.com/welcome/add-new-site/>`_ for the most up-to-date and
```

Comment:
This seems like a dup of the link below. Is this the incorrect link, or could we possibly remove one of these links if they're both going to the same destination?

Reply:
I can't remember if we added this link in or the Netlify team did, but if this is bad practice, we can either remove or change the link here to https://docs.netlify.com/core/overview/.
To be fair, it seems like the link for Vercel in the 'Important' text box is also duplicated later on in that migration guide for "visit the official Vercel docs."

Reply:
I believe I added the links to the `important` admonition. Doesn't seem necessary now.